lib/matplotlib/axes/_axes.py

```diff
@@ -5334,6 +5334,7 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
             define the data range that the colormap covers. By default,
             the colormap covers the complete value range of the supplied
             data. It is deprecated to use *vmin*/*vmax* when *norm* is given.
+            When using multichannel data, parameters *vmin*/*vmax* are ignored.
```

Comment:
```suggestion
            When using RGB(A) data, parameters *vmin*/*vmax* are ignored.
```

I wonder if we should also warn on that case, since they are invalid kwargs oil the data is 3/4-d

Reply:
Agree about warning, but not going to hold this PR over that.